qiskit/circuit/library/standard_gates/ms.py

```diff
@@ -31,6 +33,9 @@ class MSGate(Gate):
     def __init__(self, num_qubits, theta, *, n_qubits=None,  # pylint:disable=unused-argument
                  label=None):
         """Create new MS gate."""
+        warnings.warn('`MSGate` has been deprecated. Please use `GMSGate` '
```

Comment:
Could you update the text to adhere to the deprecation policy text? https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-warnings

Reply:
Updated in https://github.com/Qiskit/qiskit-terra/pull/4979/commits/56464ce6fa7dad390163b2169b934eb469dad793.